### PR TITLE
My Requests View

### PR DESCRIPTION
**Pull request by vadnguyen**
Updated Layout from list to table format
Updated UI Styling: added button scss component, added view specific styling to global application.scss
NOTE ** trouble with connecting to get list of requests for 'Test' user. Still wrapping trying to catch up

Screen Shot 2020-04-02 at 1 22 57 PM

### CODE REVIEW

File: src/components/MyRequestsView.js [hunk 1]
```diff
@@ -37,40 +38,46 @@ class MyRequestsView extends Component {
...
-            <h1>Your Requests</h1>
-          </div>
-          <div className="c-profile">
-            <button className="c-button" onClick={this._createNewRequest}>
-              Create New Request
-            </button>
-          </div>
-        </div>
-        <div className="c-requests container">
+      <div className="my-requests">
```
Comment (ConnorDY) [inline src/components/MyRequestsView.js:41]: Can you fix the indentation here?

File: src/components/MyRequestsView.js [hunk 2]
```diff
@@ -37,40 +38,46 @@ class MyRequestsView extends Component {
...
+      <Row className="view-header">
+        <Col>
+          <h1 className="h1">Your Requests</h1>
+        </Col>
+
+        <Col className="download-models">
```
Comment (ConnorDY) [inline src/components/MyRequestsView.js:47]: We should rename this class since this name doesn't apply here.

Comment (vadnguyen) [reply src/components/MyRequestsView.js]: updated

File: src/components/MyRequestsView.js (hunk 1 above)
Comment (vadnguyen) [reply src/components/MyRequestsView.js:41]: fixed

**Comment by ConnorDY** [comment]
@vadnguyen Please fix merge conflicts then merge this PR.

**Pull request merged**